tests/test_zipper.py

```diff
@@ -61,14 +61,14 @@ def test_replace_handler_in_zip(temp_dir, app_zip):
         proc_name="hello",
         proc_signature="()",
         handler="app.hello",
-        zip_file_path=app_zip,
+        zip_file_path=str(app_zip),
         coverage_reports_stage="@example",
         coverage_reports_stage_path="test_db.public.example",
     )
     assert os.path.isfile(app_zip)
     assert result == "snowpark_coverage.measure_coverage"
 
-    with ZipFile(app_zip, "r") as zip:
+    with ZipFile(str(app_zip), "r") as zip:
```

Comment:
Why not `Path`?

Reply:
`Expected type 'Union[str, PathLike[str]]', got 'Generator' instead` warning in IDE, string solves the issue